test-requirements.txt

```diff
@@ -1,4 +1,11 @@
 fixtures>=1.3.1
 hacking>=0.10,<0.11
 mock>=1.0.1,<1.1.0
+mox
+mox3
 nose
+nose-exclude
+
+# TODO(dougwig) -- can we remove this?
+-e git+https://github.com/openstack/neutron#egg=neutron
```

Comment:
If we mock the whole neutron module in tests, I believe so.  It appears the only reason it's not is we're using constants from neutron.